source/jormungandr/tests/tests_mechanism.py

```diff
@@ -530,3 +539,10 @@ def mock_car_park_providers(pois_supported):
         new_callable=mock.PropertyMock,
         return_value=lambda: providers,
     )
+
+
+def mock_equipment_providers(equipment_provider_manager, data, code_types_list):
+    equipment_provider_manager._equipment_providers = {
+        "sytral": SytralProvider(url="fake.url", timeout=3, code_types=code_types_list)
+    }
+    equipment_provider_manager._equipment_providers["sytral"]._call_webservice = MagicMock(return_value=data)
```

Comment:
minor: I prefer the usage of https://pypi.org/project/requests-mock/ for mocking webservices

Reply:
good to know